src/game.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+class Game {
+  constructor() {
+    this.frame = null;
+    this.turn = 0;
+    this.scorecard = [];
+    this.MAXIMUM_FRAME = 10;
+  }
+
+  beginFrame() {
+    if(this.turn >= this.MAXIMUM_FRAME) {
+      return this.finishGame();
+    } 
+    this.turn += 1;
+    this.frame = new Frame(this.turn);
+    return `Frame ${this.turn}`;
+  }
+
+  throwBall(points) {
+    return this.frame.addRoll(points);
+  }
+
+  addToScorecard() {
+    this.scorecard.push(this.frame.score);
+    return this.scorecard;
+  }
+
+  strikeBonus() {
+    if(this.isLastFrame()) {
+      return null;
+    }
+    if(this.scorecard[this.scorecard.length -2] === [10] || [0, 10]) {
```

Comment:
Are you sure this line is doing what you want it to do? The `===` operator takes precedence over `||`, so what you wrote there is the equivalent to doing:

`(this.scorecard[this.scorecard.length -2] === [10]) || [0, 10]`

What will be the result of this expression if `(this.scorecard[this.scorecard.length -2] === [10])` is false?